src/neo/Ledger/Blockchain.cs

```diff
@@ -28,7 +29,9 @@ public class FillMemoryPool { public IEnumerable<Transaction> Transactions; }
         public class FillCompleted { }
         internal class PreverifyCompleted { public Transaction Transaction; public VerifyResult Result; public bool Relay; }
         public class RelayResult { public IInventory Inventory; public VerifyResult Result; }
+        private class UnverifiedBlocks { public LinkedList<Block> Blocks = new LinkedList<Block>(); public HashSet<IActorRef> Nodes = new HashSet<IActorRef>(); public int Size; }
 
+        private const int MaxUnverifiedBlockSize = 100 * 1024 * 1024;
```

Comment:
Can this be removed?

Reply:
If we don't control the size of this list, we can put a lot of different blocks with the same height with different nodes

Reply:
But the number of remote nodes can be limited.